.github/workflows/fetch.yml

```diff
@@ -359,34 +357,27 @@ jobs:
         run: go install ./cmd/vuls-data-update
 
       - name: Fetch
-        run: vuls-data-update fetch windows-msuc --dir vuls-data-raw-windows-msuc.${{ strategy.job-index }} ${{ matrix.target }}
+        run: vuls-data-update fetch windows-msuc --dir vuls-data-raw-windows-msuc ${{ matrix.target }}
+
+      - name: Create tarball
+        run: tar --remove-files -acf vuls-data-raw-windows-msuc.tar.zst vuls-data-raw-windows-msuc
 
-      - name: Save cache
-        uses: actions/cache/save@v4
+      - name: Login to GitHub Packages Container registry
+        uses: docker/login-action@v3
         with:
-          path: vuls-data-raw-windows-msuc.${{ strategy.job-index }}
-          key: vuls-data-raw-windows-msuc-${{ strategy.job-index }}-${{ github.sha }}-${{ github.run_id }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-  generate-msuc-cache-id:
-    name: Generate vuls-data-raw-windows-msuc cache id
-    runs-on: ubuntu-latest
-    needs: fetch-msuc
-    outputs:
-      cid: ${{ steps.generate.outputs.cid }}
-    steps:
-      - name: Generate cache id
-        id: generate
-        run: echo "cid=$(seq 0 ${{ needs.fetch-msuc.outputs.total }} | jq -R -s -c 'split("\n")[:-2]')" >> $GITHUB_OUTPUT
+      - name: Upload vuls-data-raw-windows-msuc-data-${{ strategy.job-index }} tarball to GHCR
+        run: |
+          oras push ghcr.io/vulsio/vuls-data-db:vuls-data-raw-windows-msuc-data-${{ strategy.job-index }} vuls-data-raw-windows-msuc.tar.zst:application/vnd.vulsio.vuls-data-db.data.layer.v1.tar+zstd
```

Comment:
Is it nice to include "tmp" or a similar word to tag?

Reply:
I chose "data" because I thought it would be easier to understand that data-1 will be the final data that is aggregated, rather than "tmp-1."

Reply:
I interpreted it as temporary data and used the "data" part instead of "tmp"

Reply:
https://github.com/vulsio/vuls-data-db/compare/1ad042e2401579a79ddd103a6ff2d2056298485d..538f2dab06662c6261bbcfec52aefdae7e04d4f0

Reply:
Great!